interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is the syntax used in React, like a hybrid between HTML and Javascript. One syntax difference would be the need for {} in JSX. 
 
-Researched answer:
+Researched answer: JSX is an extension of the Javascript language syntax. In JSX you must return a single parent element.
 
-1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
+1. What is yarn? What file(s) are modified when you run the command yarn in your terminal? 
 
-Your answer:
+Your answer: Yarn is a package manager. Testing files are modified.
 
-Researched answer:
+Researched answer: Yarn is a package manager for your code. Yarn modified j.son files.
 
 3. What is a React component?
 
-Your answer:
+Your answer: A React component is a piece of code branching out from the App.js component. 
 
-Researched answer:
+Researched answer: An independent reusable bit of code which divides the UI into smaller pieces.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: State values are those who are modified with a given input. Props are in a sense a shortcut to access certain values into a component.
 
-Researched answer:
+Researched answer: State is internal and controlled by the component itself while props are external and controlled by whatever renders the component
 
 5. What is the DOM?
 
-Your answer:
+Your answer: Is the Document Object Model, and its the structure for code. 
 
-Researched answer:
+Researched answer: The Document Object Model defines the logical structure of documents and the way the document is accessed and manipulated.
 
 6. STRETCH: Which is the difference between a div tag and a span tag?
 
-Your answer:
+Your answer: A div tag set for a new block, where code can be. Kind of like a container. While a span, i have no idea. 
 
-Researched answer:
+Researched answer: A div tag creates a block level element, while the span tag wraps around an inline element. 
```

Comment:
These interview questions could use some more elaboration. Most of these answers do not fully explain/answer what the question was asking. Do some more research so you can benefit from having a deeper understanding of these concepts.